### PR TITLE
Feature/add geog hierarchy to key

### PR DESCRIPTION
**Pull request by fenallen**
### What

Refactors tests to allow loading of selected standing data.
Adds test that loads multiple geographic hierarchies.
Turns on ArmedForces loading test by default.

### How to review

sbt clean compile test
sbt it:test

### Who can review

Anyone

### CODE REVIEW

File: it/main/LoadArmedForcesSampleCsvToDatabaseTest.java [hunk 1]
```diff
@@ -20,30 +16,20 @@
...
+    private EntityManager em;
+    private PostgresTest postgresTest = new PostgresTest();
 
     @BeforeClass
     public void setupJPA() {
-
         logger.info("SETTING UP JPA");
-        final Map<String, Object> databaseParameters = Configuration.getDatabaseParameters();
-        emf = Persistence.createEntityManagerFactory("data_discovery", databaseParameters);
+        emf = postgresTest.getEMFForProducitonLikeDatabase();
```
Comment (NeilMadden) [inline it/main/LoadArmedForcesSampleCsvToDatabaseTest.java]: Pedantic Typo Alert: Produciton -> Production

File: it/main/LoadMultipleGeographicalHierarchiesTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+            // TODO sort the results properly based on hierarchy once hierachyBean nonsense is deleted - this temporarily just gives a consistent order
+            results = results.stream().sorted(Comparator.comparing(GeographicArea::getGeographicAreaId)).collect(Collectors.toList());
+
+            assertEquals(results.get(0).getGeographicAreaHierarchyBean().getGeographicAreaHierarchy(), "2011GPH");
+            assertEquals(results.get(1).getGeographicAreaHierarchyBean().getGeographicAreaHierarchy(), "2013ADMIN");
+
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail();
```
Comment (NeilMadden) [inline it/main/LoadMultipleGeographicalHierarchiesTest.java:60]: Won't letting the exception propagate do the same thing?

**Review by NeilMadden — APPROVED** [review]
A couple of minor points. Just testing with my changes too to make sure no conflicts.

**Comment by NeilMadden** [comment]
Double-checked that there is no conflict with my changes: I can merge your changes in with no conflicts and all tests and it:test pass.

File: it/main/LoadMultipleGeographicalHierarchiesTest.java (hunk 2 above)
Comment (fenallen) [reply it/main/LoadMultipleGeographicalHierarchiesTest.java:60]: You might think so, but it fails to compile without it.

**Pull request merged**